Library/Formula/gcviewer.rb

```diff
@@ -0,0 +1,14 @@
+class Gcviewer < Formula
+  homepage "https://github.com/chewiebug/GCViewer"
+  version "1.34.1"
```

Comment:
Merged this directly into the `url`. Homebrew isn't a huge fan of variables in `url`s. Just ` url "https://github.com/chewiebug/GCViewer/archive/1.34.1.tar.gz" ` or similar, Thanks!